network.go

```diff
@@ -230,6 +231,21 @@ func (s *sandbox) removeInterface(netHandle *netlink.Handle, iface *types.Interf
 	return nil, nil
 }
 
+func expectLinkUpdate(ch <-chan netlink.LinkUpdate, ifaceName string) bool {
+	for {
+		timeout := time.After(time.Second * 5)
+		select {
+		case update := <-ch:
+			// Reference: https://www.kernel.org/doc/Documentation/networking/operstates.txt
+			if ifaceName == update.Link.Attrs().Name && (update.IfInfomsg.Flags&unix.IFF_UP != 0) && update.Link.Attrs().OperState == netlink.OperUp {
```

Comment:
Could you reformat this line as it makes my eyes ache? :smile:

Reply:
haha :) I'll try to see how I can simplify it.